examples/web-ui/material/menus/menuDesktop.smash

```diff
@@ -0,0 +1,9 @@
+* Set Desktop
+    * Click first level page button {{page}}
+        Log "Desktop click"
```

Comment:
I wouldn't log anything. The report will already contain the desktop and click steps above.